plugins/CircuitExplorer/plugin/io/morphology/neuron/NeuronBuilder.h

```diff
@@ -75,3 +75,42 @@ class NeuronBuilder
 private:
     const std::string _name;
 };
+
+/**
+ * @brief The NeuronBuilderTable class register and gives access to all
+ * available neuron builders, identifying them by their name
+ */
+class NeuronBuilderTable
+{
+public:
+    /**
+     * @brief NeuronBuilderTable constructor registers all available builders
+     */
+    NeuronBuilderTable();
+
+    /**
```

Comment:
Just a personal preference but usually I write it like that:
```
void registerBuilder(std::unique_ptr<NeuronBuilder> builder) { // register builder }

template<typename T, typename ...Args>
void register(Args &&...args)
{
    registerBuilder(std::make_unique<T>(std::forward<Args>(args)...));
}
```
Then you can register builders with no default constructors and it is always convenient to have a non-template method to do the registering (and the nasty SFINAE is not required as the code would not compile if T is not a Builder anyway).